makefiles/Makefile.docs-404

```diff
@@ -1,32 +1,23 @@
 GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
-
+USER=$(shell whoami)
 SNOOTY_ENV = $(shell printenv SNOOTY_ENV)
-
-
 BUCKET = $(shell printenv BUCKET)
-URL = $(shell printenv URL)
 
-ifeq ($(STAGING_USERNAME),)
-	USER=$(shell whoami)
-else
-	USER=$(STAGING_USERNAME)
-endif
 REPO_DIR=$(shell pwd)
 SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
 SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
 PROJECT=docs-404
 MUT_PREFIX=404
 CUSTOM_NEXT_GEN_DEPLOY=true
 
-
-ifeq ($(SNOOTY_ENV), production) 
-	DEPLOY_JOBS=docs-mongodb-org-prd
-else ifeq ($(SNOOTY_ENV), staging)
-	DEPLOY_JOBS=docs-mongodb-org-stg
-else ifeq ($(SNOOTY_ENV), regression)
-	DEPLOY_JOBS=docs-mongodb-org-intgr
+ifeq ($(SNOOTY_ENV), dotcomstg)
+	EXECUTE_DEPLOY=true
+else ifeq ($(SNOOTY_ENV), dotcomprd)
+	EXECUTE_DEPLOY=true
 endif
 
+DEPLOY_JOBS=job-$(BUCKET) job-docs-atlas-$(SNOOTY_ENV) job-docs-cloudmanager-$(SNOOTY_ENV) job-docs-opsmanager-$(SNOOTY_ENV) job-docs-java-$(SNOOTY_ENV) job-docs-go-$(SNOOTY_ENV) job-docs-node-$(SNOOTY_ENV)
```

Comment:
Flagging here that Cloud Manager and Ops Manager both seem to be publishing to the regular dotcomprd bucket based on [this Makefile](https://github.com/10gen/mms-docs/blob/master/makefile#L51-L53) and the "Last Modified" dates of the files in the buckets. It might make sense to omit `job-docs-cloudmanager-$(SNOOTY_ENV)` and `job-docs-opsmanager-$(SNOOTY_ENV)` if we don't need them

Reply:
Nice catch! Confirmed in Compass that neither `{ "bucket.dotcomprd": "docs-opsmanager-dotcomprd" }` nor `{ "bucket.dotcomprd": "docs-cloudmanager-dotcomprd" }` return any results. I'll update the Makefiles.

For the sake of cleanup, is it worth making a backlog ticket to clean up our Cloud Formation template `buckets.yml`? I think it'd be good to maintain the our Cloud Formation template as a source of truth for buckets we're actively using to publish and stage content